packages/terra-search-field/src/terra-dev-site/doc/search-field/SearchField.1.doc.jsx

```diff
@@ -21,7 +21,7 @@ import SearchFieldDefaultValueSrc from '!raw-loader!../../../../src/terra-dev-si
 import SearchFieldDisabled from '../example/SearchFieldDisabled';
 import SearchFieldDisabledSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/SearchFieldDisabled';
 import SearchFieldDisabledAutoFormatting from '../example/SearchFieldDisableAutoFormatAttrs';
-import SearchFieldDisabledAutoFormattingSrc from '../example/SearchFieldDisableAutoFormatAttrs';
+import SearchFieldDisabledAutoFormattingSrc from '!raw-loader!../../../../src/terra-dev-site/doc/example/SearchFieldDisableAutoFormatAttrs';
```

Comment:
do we need this change ?

Reply:
Yes. Currently the official website does not show the sample code for `Disabled Auto Formatting`. This change fixes that.